tools/ci_build/github/azure-pipelines/c-api-packaging-pipelines.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     - template: templates/set-test-data-variables-step.yml
     - template: templates/set-version-number-variables-step.yml
 
-    - script: 'tools/ci_build/github/linux/run_dockerbuild.sh -o ubuntu16.04 -d cpu -r $(Build.BinariesDirectory) -x " --test_data_url $(TestDataUrl) --test_data_checksum $(TestDataChecksum)"'
+    - script: 'tools/ci_build/github/linux/run_dockerbuild.sh -o ubuntu16.04 -d cpu -r $(Build.BinariesDirectory) -x "--build_wheel --test_data_url $(TestDataUrl) --test_data_checksum $(TestDataChecksum)"'
```

Comment:
build_wheel can be left off for c api packaging.

Reply:
Done